mitremap-notebook/model.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Change of plan here - Instead of letting the user choose, I am planning on presenting only one option [distilgpt2-512 model], since it has the best performance on the test data. I can talk a bit more about the distilgpt2-512 model in the readme.

Reply:
Resolved